pkg/tsdb/elasticsearch/response_parser.go

```diff
@@ -101,10 +110,11 @@ func parseResponse(ctx context.Context, responses []*es.SearchResponse, targets
 			trimDatapoints(queryRes, target)
```

Comment:
What do you think about adding another `span` error handling in lines 105 to 107?

Reply:
👍 Added in https://github.com/grafana/grafana/pull/74750/commits/1509a01bb63bc85fccb069bdb2350ed20475f40a